crates/consensus/beacon/src/engine/sync.rs

```diff
@@ -317,8 +317,8 @@ where
             let mut request = self.inflight_block_range_requests.swap_remove(idx);
             if let Poll::Ready(blocks) = request.poll_unpin(cx) {
                 trace!(target: "consensus::engine", len=?blocks.len(), first=?blocks.first().map(|b| b.num_hash()), last=?blocks.last().map(|b| b.num_hash()), "Received full block range, buffering");
-                self.range_buffered_blocks
-                    .extend(blocks.into_iter().map(OrderedSealedBlock).map(Reverse));
+                    self.range_buffered_blocks
```

Comment:
hi @mattsse could you help me on this part, I can't figure out how to fix the compile error here after introducing the generic to OrderedSealedBlock. Thank you so much!